samples/boards/esp32/flash_memory_mapped/README.rst

```diff
@@ -1,4 +1,8 @@
-.. _flash_memory_mapped:
```

Comment:
great call! :) 
But then please remove lines 7-9 as the title is auto-added by the code-sample thingy.